testutil/testutil.go

```diff
@@ -0,0 +1,45 @@
+package testutil
+
+import (
+	"time"
+
+	"github.com/evergreen-ci/logkeeper/db"
+	"github.com/evergreen-ci/logkeeper/env"
+	"github.com/pkg/errors"
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+func InitDB() error {
+	connInfo := mgo.DialInfo{
+		Addrs:   []string{"localhost"},
+		Timeout: 5 * time.Second,
+	}
+	session, err := mgo.DialWithInfo(&connInfo)
+	if err != nil {
+		return errors.Wrap(err, "connecting to db")
+	}
+
+	if err = env.SetSession(session); err != nil {
+		return errors.Wrap(err, "setting session")
+	}
+
+	env.SetDBName("logkeeper_test")
+
+	return nil
+}
+
+// ClearCollections clears all documents from all the specified collections,
+// returning an error immediately if clearing any one of them fails.
+func ClearCollections(collections ...string) error {
+	db, closer := db.DB()
+	defer closer()
+
+	for _, collection := range collections {
```

Comment:
If we wanted to be paranoid we could verify that the db we are talking to is "logkeeper_test" (to avoid accidentally deleting a real DB if an environment was misconfigured or something.

Reply:
I tried a different strategy where we set a build tag during testing that will create a connection to the db and implement this function while production code doesn't implement the function. Apparently [gofmt](https://go.dev/doc/go1.17#gofmt) adds both types of build tags so it will be forwards and backwards compatible.

Does this sound like a good idea? One effect that occurs to me is that if a developer isn't using the makefile to run tests (like using VSCode's "Run Test" button) they'll need to set the `test` build flag (in workspace settings for vscode).

Reply:
I usually prefer to be explicit about setup but I think that matches how we do that in evergreen.

Reply:
I've backed out the changes to use a build tag for this.
I restored the explicit call to `InitDB()`